armsrc/lfops.c

```diff
@@ -1203,6 +1203,8 @@ void CmdIOdemodFSK(int findone, int *high, int *low, int ledcontrol)
 #define WRITE_0   18*8 // was 144 // SPEC: 16*8 to 32*8 - typ 24*8 (or 24fc)
 #define WRITE_1   50*8 // was 400 // SPEC: 48*8 to 64*8 - typ 56*8 (or 56fc)  432 for T55x7; 448 for E5550
 #define READ_GAP  15*8 
+// Long Leading Reference
+#define Reference_llr (136+18)*8  // Needs to be WRITR_0 + 136 clocks.
```

Comment:
If this needs to e WRITE_0 + 136 clocks, then let's code it as
```
#define Refrence_llr (WRITE_0 + 136) * 8

Reply:
Yes, good point.  
It will need to become
#define Refrence_llr ((WRITE_0/8) + 136) * 8
i.e. pre * 8.

Reply:
or 
`#define Reference_llr (WRITE_0 + 136 * 8)`

Reply:
I have setup a data structure to hold the timings for each downlink mode.  This will allow a future addition to set the timings from the client.  Since (atm) the LLR reference should be WRITE_0 +136 clocks, I have removed the storage of that and calculate it when used (forcing it to be correct if the timings are changed from the client.